README.md

```diff
@@ -19,20 +22,27 @@ running. Check if you have to run docker as local user or as root.
 
     $ docker info
 
-Then run:
+To setup the instance directory run:
 
     $ ./openslides setup .
+
+Before starting the instance you must setup a means of SSL encryption because
+OpenSlides only runs over HTTPS. See [HTTPS](#HTTPS)
```

Comment:
I think it would be nice if there was a step by step description. If you are reading this, you have to jump to the Https chapter. So why not describing it here.

Reply:
I kinda agree with you and at first I was writing it that way.
However I think it does clutter the guide quite a bit, as its not 2 more straight steps to do to complete the setup. Rather it exhibits options a user has.
So I think of it like "Step 3: look at HTTPS options and do what fits your environment"
Also its about editing the `config.yml` so that was another reason for me to have it under the configuration section

Reply:
Maybe we could make it easier.

The manage server could generate self signed certs and automatically use them. This in quite easy with go.

Then it would only be one step like 

openslides setup --create-certs

Reply:
I was thinking leaving the choice of cert creation to the user would be better.
But this might be a nice optional feature.
In the step-by-step setup guide `openslildes setup --create-certs` could be listed as the easiest/fastest way to have localhost HTTPS. Additionally the HTTPS section can be linked for use cases where different certs a needed.

> The manage server could generate self signed certs and automatically use them. This in quite easy with go.
@normanjaeckel is this doable for you?

Reply:
Ok for me.